Taskfile.yml

```
@@ -312,3 +312,4 @@ tasks:
           PRIVATE_KEY: "0000000000000000000000000000000000000000000000000000000000000001"
           PROVIDER: "http://localhost:8484"
           ADMIN_WALLET: "{{.DEV_DB_OWNER}}"
+      - kwil-cli exec-action grant_roles text:system text:network_writer text[]:{{.DEV_DB_OWNER}} --private-key 0000000000000000000000000000000000000000000000000000000000000001 --provider http://localhost:8484 --sync
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)